src/taipy/gui/gui.py

```diff
@@ -734,10 +734,10 @@ def __send_ws(self, payload: dict) -> None:
                     payload,
                     to=self.__get_ws_receiver(),
                 )
-                time.sleep(0)
+                time.sleep(0.001)
```

Comment:
:-o

Reply:
0.001 is the default. If I dont put it then a warning will be shown in gevent 22 :D